pyopia/statistics.py

```diff
@@ -38,15 +41,19 @@ def d50_from_vd(vd, dias):
     '''
     Calculate d50 from a volume distribution
 
-    Args:
-        vd            : particle volume distribution calculated from vd_from_stats()
-        dias          : mid-points in the size classes corresponding the the volume distribution,
-                               returned from get_size_bins()
+    Parameters
+    ----------
+    vd : array
```

Comment:
Further down, in another function, ```volume_distribution``` is used as variable name instead of ```vd```. Maybe go consistently for the longer name?